### PR TITLE
New slack.json - single hook

### PR DESCRIPTION
**Pull request by filmaj**
This PR builds off of @sopokuagyemang's #20 PR but uses the now-public `deno_slack_hooks` repository to implement the single-hook approach (all your commits are still here @sopokuagyemang - I just changed slack.json in my last commit).

This PR relies on @sopokuagyemang's https://github.com/slackapi/slack-cli/pull/477 PR, and until that is merged into `main`, this PR probably should not be merged either. Until that time, you will have to use that specific slack-cli branch in order to use this version of the app template.

To get started:

    slack create -t slackapi/deno-reverse-string -b new-slack-json-one-hook
    cd monstrous-gerbil-123

Then, things you can test (that I tested successfully using the above slack-cli PR and this branch of the template):

1. Deploying. Try e.g. `slack deploy --slackdev --runtime deno1.19`.
2. Uninstalling a deployed app, e.g. `slack workspace uninstall --slackdev`.
3. Running locally, e.g. `slack run --slackdev --runtime deno1.19`. While it is running, you can also:
  - Change some core property inside `manifest.ts`. Say, the name of the app. The CLI should detect the change and redeploy - the app should be renamed shortly after you save the change to `manifest.ts` and within a few seconds should be reflected on your workspace.
  - I was also able to change where the function code lives. E.g. `mv functions funks`, then update the `source_file` property inside `manifest.ts` from `functions/reverse.ts` to `funks/reverse.ts`. After the CLI detects the change, try running your function - it should work.
  - Introducing a syntax error in your function code, then re-running the function. It should not work, but you should get a meaningful error in your `slack run` terminal output, related to your syntax error.

Please test other things !

### CODE REVIEW

**Review by mwbrooks — APPROVED** [review]
Thanks for the test instructions. I've tested this using PR https://github.com/slackapi/slack-cli/pull/477 and it works. Tested:
* Local run
* Changes files with watch
* Deploy
* Uninstall

File: slack.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
+    "get-hooks": "deno run -q --unstable --allow-read --allow-net https://deno.land/x/deno_slack_hooks@0.0.2/mod.ts"
```
Comment (shapirone) [inline slack.json]: `0.0.1` is the latest release for the hooks lib, but I did just need to bump the version of the builder. Are we ready to cut and use `0.0.2` of the hooks lib? May want to merge https://github.com/slackapi/deno-slack-hooks/pull/6/files first

Comment (mwbrooks) [reply slack.json]: It looks like [deno_slack_hooks@0.0.2](https://deno.land/x/deno_slack_hooks) was released yesterday, but we could do another release that adds PR https://github.com/slackapi/deno-slack-hooks/pull/6?

Comment (filmaj) [reply slack.json]: @mwbrooks yep, whatever we need! Tagging and releasing is cheap and fast so let's do it as much as we need to!

**Pull request merged**